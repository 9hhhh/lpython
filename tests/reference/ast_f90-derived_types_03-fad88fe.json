{
    "basename": "ast_f90-derived_types_03-fad88fe",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/derived_types_03.f90",
    "infile_hash": "fd0a87db249f310cd20a90b39d5eb4627597acf774b62bdb8c4256c4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-derived_types_03-fad88fe.stdout",
    "stdout_hash": "fcf4d2e46bed4c4ba969f402333c6801cf41dbd08cb8c6e8b704bb82",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}