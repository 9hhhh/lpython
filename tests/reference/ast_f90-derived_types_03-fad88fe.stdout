program derived_types_03
type :: X
    integer :: i
end type
type :: b

contains

    subroutine Y()
    type :: A
        integer :: i
    end type
    type :: b
    end subroutine Y

    integer function Z() 
    type :: A
        integer :: i
    end type
    type :: b
    Z = 5
    end function Z

end program derived_types_03
